Makefile

```
@@ -44,5 +44,11 @@ uiduid: uiduid.m ent.plist
 uiopen: uiopen.m ent.plist
 	$(CC) -fobjc-arc -O3 $(CFLAGS) uiopen.m -o uiopen $(LDFLAGS) -framework Foundation -framework MobileCoreServices
 
+localelocale: localelocale.m ent.plist
+	$(CC) -fobjc-arc -O3 $(CFLAGS) localelocale.m -o localelocale $(LDFLAGS) -framework Foundation
+
+serialserial: serialserial.m ent.plist
+	$(CC) -fobjc-arc -O3 $(CFLAGS) serialserial.m -o serialserial $(LDFLAGS) -framework Foundation -lMobileGestalt
```

serialserial doesn't use any Foundation code so you can link with CoreFoundation instead
```suggestion
	$(CC) -fobjc-arc -O3 $(CFLAGS) serialserial.m -o serialserial $(LDFLAGS) -framework CoreFoundation -lMobileGestalt
```